### ISSUE TITLE
Keyboard Accessibility/shortcuts :: Ensure that all content can be accessed with the keyboard alone

### ISSUE THREAD

**Issue by SteadyCadence** [Cadasta/cadasta-qgis-plugin#175]
Not sure what QGIS's support with those with disabilities are, but I would like the plugin to support accessibility requirements.  You should be able to use `tab` and `enter` to go through the plugin windows. 

http://webaim.org/techniques/keyboard/

**Comment by cchristelis** [comment]
Hi @SteadyCadence, I think this is a great suggestion. @dimasciput, there is a tab ordering​ view in QtCreator.

**Comment by cchristelis** [comment]
Done in #188
